### DISCUSSION TITLE
Cannot trigger dispatches from GitHub API

### DISCUSSION THREAD

**Discussion by zhang35** [community/community#74542] [Actions]
### Select Topic Area

Bug

### Body

I want to trigger a GitHub Actions workflow manually:
https://github.com/zhang35/test-bash-jiaqi/actions/workflows/jiaqi-test.yml

Through the docs: https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#create-a-workflow-dispatch-event I found out I can do so through a workflow dispatch event.

curl -L \
  -X POST \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer xxx" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/zhang35/test-bash-jiaqi/actions/workflows/jiaqi-test/dispatches \
  -d '{"ref":"topic-branch","inputs":{"name":"Mona the Octocat","home":"San Francisco, CA"}}'

The problem is that when I hit the API at the /dispatches endpoint, I get the following error:

```
{
    "message": "Not Found",
    "documentation_url": "https://docs.github.com/rest/actions/workflows#create-a-workflow-dispatch-event"
}
```

I have already given full permission to the token. So how can I solve this error?

**Comment by ank-ankur** [comment]
@zhang35 , i have the exact same problem, were you able to fix this?

**Reply by zhang35** [reply]
The URL was wrong! The workflow file name should include the suffix: `.yml`:

https://api.github.com/repos/zhang35/test-bash-jiaqi/actions/workflows/jiaqi-test.ym/dispatches

Then it worked!
